### DISCUSSION TITLE
Sunbird-Content-Editor - Setup

### DISCUSSION THREAD

**Discussion by kiran9ops** [Sunbird-RC/community#263] [Issues]
Hi Team,

I tried setting up the Sunbird Content Editor following below link on my local Ubuntu machine, but I am not able to set it up properly.
**https://github.com/project-sunbird/sunbird-content-editor**

After final step of installation, enable 3000 port and able to access the link - http://localhost:3000/app

http://localhost:3000/app?contentId={{content_id}}

I am not able to figure out the content_id where I can get it from and the configurations file what all I need to update to access the link

Can you please help me out to resolve this configuration issue.

**Comment by kiran9ops** [comment]
Can you please guide us to resolve the issue

**Reply by coolbung** [reply]
Hi. This is the Sunbird RC Forum.

The sunbird content editor is part of the Sunbird ED / coKreat project. 

If you are adopting Sunbird ED, then you may ask in the ED discussions - https://github.com/orgs/Sunbird-Ed/discussions
Or if you're trying out the editor only, then in coKreat - https://github.com/orgs/Sunbird-coKreat/discussions

**Comment by vinukumar-vs** [comment]
Thanks for reaching out. Can you please move this discussion to below Building Block
https://github.com/Sunbird-Knowlg/Community/discussions/

We will help you in setting the [content-editor](https://github.com/project-sunbird/sunbird-content-editor/tree/release-5.0.0)
